doc/api/process.markdown

```diff
@@ -803,6 +803,35 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.resourceUsage();
+
+Returns the resource usage measures
+
+Note: Not all resource usage measures are available on Windows. Fields
```

Comment:
Do you know exactly which fields aren't available on Windows? If so, you could call them out.